modules/core/src/main/scala/gem/config/Gmos.scala

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.config
+
+import gem.enum._
+
+import java.time.Duration
+
+/** Additional type hierarchy over the low-level GMOS enums. */
+object Gmos {
+
+  final case class GmosNodAndShuffle(
+    /*placeholder for now*/
+  )
+
+  /** Shared static configuration for both GMOS-N and GMOS-S.
+    */
+  final case class GmosCommonStaticConfig(
+    detector:      GmosDetector,
+    mosPreImaging: Boolean,
+    nodAndShuffle: Option[GmosNodAndShuffle]
+  )
+
+  object GmosCommonStaticConfig {
+    val Default: GmosCommonStaticConfig =
+      GmosCommonStaticConfig(
+        GmosDetector.HAMAMATSU,
+        false,
+        None
+      )
+  }
+
+  /** Parameters that determine GMOS CCD readout.
+    */
+  final case class GmosCcdReadout(
+    xBinning:    GmosBinning,
+    yBinning:    GmosBinning,
+    ampCount:    GmosAmpCount,
+    ampGain:     GmosAmpGain,
+    ampReadMode: GmosAmpReadMode
+  )
+
+  object GmosCcdReadout {
+    val Default: GmosCcdReadout =
+      GmosCcdReadout(
+        GmosBinning.One,
+        GmosBinning.One,
+        GmosAmpCount.Twelve,
+        GmosAmpGain.Low,
+        GmosAmpReadMode.Slow
+      )
+  }
+
+  /** Shared dynamic configuration for both GMOS-N and GMOS-S.
+    */
+  final case class GmosCommonDynamicConfig(
+    ccdReadout:   GmosCcdReadout,
+    dtaxOffset:   GmosDtax,
+    exposureTime: Duration
+  )
+
+  object GmosCommonDynamicConfig {
+    val Default: GmosCommonDynamicConfig =
+      GmosCommonDynamicConfig(
+        GmosCcdReadout.Default,
+        GmosDtax.Zero,
+        Duration.ofSeconds(300)
+      )
+  }
+
+  /** Custom mask definition, which is available as an alternative to using a
+    * builtin FPU.  Either both these parameters are set or neither are set in a
+    * GMOS observation
+    */
+  final case class GmosCustomMask(
+    maskDefinitionFilename: String,
+    slitWidth:              GmosCustomSlitWidth
+  )
+
+  /** GMOS grating central wavelength.  For now, just a value class wrapper
+    * around a Double.  This should be switched to Fixed / squants?
+    */
+  final case class GmosCentralWavelength(val nm: BigDecimal) extends AnyVal
```

Comment:
Comment is incorrect. `BigDecimal` seems good. We should look at [Libra](https://to-ithaca.github.io/libra/) for dimensions … it may end up being better than Squants.